.azuredevops/release.yml

```diff
@@ -113,14 +113,12 @@ extends:
             inputs:
               command: restore
               feedsToUse: select
-              vstsFeed: $(VSTS_FEED_ID)

```

Comment:
I discovered you can provide the name rather than an ID. This is makes things much simpler since we don't need to hide away an internal ID anymore.